tests/kubeless-test-client/README.md

```diff
@@ -1,9 +1,7 @@
 The `kubeless-test-client` contains the following:
-* A `Dockerfile` for the image used in Kyma kubeless tests
-* A go file named test-kubeless.go, which executes the tests for the kubeless chart
-
-Use the following command to build image:
-
-```bash
-$ make build-release-image RELEASE_VERSION=1.6.0
-```
+* A `Dockerfile` for the image used in Kyma Kubeless tests
+* A Go file named `test-kubeless.go`, which executes the tests for the Kubeless chart
+* A namespace YAML, which specifies the `kubeless-test` namespace
+* A `k8s.yaml` file, which contains necessary resources for `test-event` and `test-hello`
+* The JavaScript files of `test-event` and `test-hello`
+* A `svcbind.yaml` file, which specifies `test-svcbind` and its supporting Kubernetes resources
```

Comment:
A little mode info ;)

Reply:
What info are you missing?

Reply:
I would something like  svcbind.yaml is creating a service biniding usage to  redis.